Makefile

```diff
@@ -3,7 +3,7 @@
 # To re-generate a bundle for another specific version without changing the standard setup, you can:
 # - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
-VERSION ?= 1.12.40
+VERSION ?= 1.12.43
```

Comment:
I wonder if we can get the version from charts/charts/sysdig/Chart.yaml, so we extract the chart version automatically?

Reply:
Agreed, that would be useful. There is a pending task to update this repo when the chart updates, so that may be a good place to hook this in